lib/common/utils.ts

```diff
@@ -50,6 +50,10 @@ export const isNode: boolean =
 export const isBrowser: boolean =
     !isNode && !isWebWorker && !!(typeof window !== 'undefined' && window['HTMLElement']);
 
+// we are in electron of nw, so we are both browser and nodejs
+export const isMix: boolean = typeof process !== 'undefined' &&
```

Comment:
I just wonder if nw.js is the same with electron, if we only handle electron, zone-electron is a better name